### ISSUE TITLE
No results with "Around a point"

### ISSUE THREAD

**Issue by coret** [Daniel-KM/Omeka-S-module-DataTypeGeometry#11]
When I search items around a point, like
![image](https://user-images.githubusercontent.com/101820/179035195-00044179-3da6-47a5-8ae5-5daa10856628.png)
I don't get any results. This is unexpected as there are a lot of items with geometry (there are 6379 records in the data_type_geometry table).

Omeka S version 3.2.1
Data Type Geometry version 3.3.1.6
MariaDB version 10.5.15

**Comment by Daniel-KM** [comment]
Thanks for your notes, they are all fixed in version 3.4.2 for Omeka S v4 (not for v3).

**Comment by Daniel-KM** [comment]
Note that MariaDB cannot give good results for geographic data (with a sphere) : it uses always a flat geometry… And this is very badly documented, only when searching for the main function : https://mariadb.com/kb/en/st_srid/ .

**Comment by Daniel-KM** [comment]
Finally, I released version 3.3.2 for Omeka S v3.

**Comment by coret** [comment]
> Finally, I released version 3.3.2 for Omeka S v3.

The [module.ini](https://github.com/Daniel-KM/Omeka-S-module-DataTypeGeometry/blob/master/config/module.ini) still says 4.0? When I manually change this to 3.2.0 the upgrade fails with the following error:
```
Error: Class 'DataTypeGeometry\DataType\Geography' not found in /home/http/goudatijdmachine.nl/omeka-s-modules/DataTypeGeometry/data/scripts/upgrade.php:89
Stack trace:
#0 /home/http/goudatijdmachine.nl/omeka-s-modules/Generic/AbstractModule.php(136): require_once()
#1 /home/http/goudatijdmachine.nl/omeka-s-3.2.3/application/src/Module/Manager.php(321): Generic\AbstractModule->upgrade()
#2 /home/http/goudatijdmachine.nl/omeka-s-3.2.3/application/src/Controller/Admin/ModuleController.php(250): Omeka\Module\Manager->upgrade()
#3 /home/http/goudatijdmachine.nl/omeka-s-3.2.3/vendor/laminas/laminas-mvc/src/Controller/AbstractActionController.php(77): Omeka\Controller\Admin\ModuleController->upgradeAction()
#4 /home/http/goudatijdmachine.nl/omeka-s-3.2.3/vendor/laminas/laminas-eventmanager/src/EventManager.php(321): Laminas\Mvc\Controller\AbstractActionController->onDispatch()
#5 /home/http/goudatijdmachine.nl/omeka-s-3.2.3/vendor/laminas/laminas-eventmanager/src/EventManager.php(178): Laminas\EventManager\EventManager->triggerListeners()
#6 /home/http/goudatijdmachine.nl/omeka-s-3.2.3/vendor/laminas/laminas-mvc/src/Controller/AbstractController.php(103): Laminas\EventManager\EventManager->triggerEventUntil()
#7 /home/http/goudatijdmachine.nl/omeka-s-3.2.3/vendor/laminas/laminas-mvc/src/DispatchListener.php(139): Laminas\Mvc\Controller\AbstractController->dispatch()
#8 /home/http/goudatijdmachine.nl/omeka-s-3.2.3/vendor/laminas/laminas-eventmanager/src/EventManager.php(321): Laminas\Mvc\DispatchListener->onDispatch()
#9 /home/http/goudatijdmachine.nl/omeka-s-3.2.3/vendor/laminas/laminas-eventmanager/src/EventManager.php(178): Laminas\EventManager\EventManager->triggerListeners()
#10 /home/http/goudatijdmachine.nl/omeka-s-3.2.3/vendor/laminas/laminas-mvc/src/Application.php(331): Laminas\EventManager\EventManager->triggerEventUntil()
#11 /home/http/goudatijdmachine.nl/omeka-s-3.2.3/index.php(21): Laminas\Mvc\Application->run()
#12 {main}
```

**Comment by Daniel-KM** [comment]
Thanks for review, there was an issue in the upgrade process. I fixed it. Use version 3.3.2-beta, not v3.4.2.
